### DISCUSSION TITLE
Are function plugins supported for graph rules?

### DISCUSSION THREAD

**Discussion by levikobi** [lf-edge/ekuiper#2131] [Q&A] [answered]
Hi,

I want to configure a plugin function that sends an HTTP request to a 3rd-party service.
From the documentation, it seems possible when using SQL rules. 
However, it doesn't seem possible when using graph rules (or is it?)

What I have seen for graph rules is an operator node with nodeType script, but I don't think it covers my use case (of calling 3rd party system via HTTP)...

Is this use case possible at the moment? Is it being worked on?
If not and this is something that you are interested in adding, I will gladly try to help.

**Comment by ngjaying** [comment, marked as answer]
Yes, there is a function node in graph rules which can call any functions include customized functions.

**Reply by levikobi** [reply]
That's awesome, thanks @ngjaying!
